### PR TITLE
Added Heat_output (Leistung Ist) to Sensors

### PR DESCRIPTION
**Pull request by Crashman1983**

### CODE REVIEW

File: custom_components/luxtronik/const.py [hunk 1]
```diff
@@ -371,6 +372,17 @@ class LuxMkTypes(Enum):
...
         sensor_key="calculations.ID_WEB_WMZ_Seit",
     ),
+        LuxtronikSensorEntityDescription(
+        key="current_heat_output",
+        # name=f"{text_heat_amount_counter}",
+        icon="mdi:lightning-bolt-circle",
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        unit_of_measurement=POWER_WATT,
+        #native_unit_of_measurement="W",
```
Comment (Kars-de-Jong) [inline custom_components/luxtronik/const.py:382]: I believe Home Assistant sensors are supposed to only use `native_unit_of_measurement` these days.

File: custom_components/luxtronik/const.py [hunk 2]
```diff
@@ -371,6 +372,17 @@ class LuxMkTypes(Enum):
         entity_category=EntityCategory.DIAGNOSTIC,
         sensor_key="calculations.ID_WEB_WMZ_Seit",
     ),
+        LuxtronikSensorEntityDescription(
+        key="current_heat_output",
+        # name=f"{text_heat_amount_counter}",
+        icon="mdi:lightning-bolt-circle",
+        device_class=DEVICE_CLASS_ENERGY,
```
Comment (Kars-de-Jong) [inline custom_components/luxtronik/const.py:379]: This should be `DEVICE_CLASS_POWER`.

File: custom_components/luxtronik/sensor.py [hunk 3]
```diff
@@ -300,6 +301,19 @@ async def async_setup_entry(
...
         ),
+        LuxtronikSensor(
+            hass,
+            luxtronik,
+            device_info,
+            sensor_key="calculations.Heat_Output",
+            unique_id="current_heat_output",
+            name=f"{text_current_heat_output}",
+            icon="mdi:lightning-bolt-circle",
+            device_class=DEVICE_CLASS_ENERGY,
```
Comment (Kars-de-Jong) [inline custom_components/luxtronik/sensor.py:312]: This should be `DEVICE_CLASS_POWER`.

File: custom_components/luxtronik/sensor.py [hunk 4]
```diff
@@ -300,6 +301,19 @@ async def async_setup_entry(
...
+            state_class=STATE_CLASS_MEASUREMENT,
+            unit_of_measurement=POWER_WATT,
```
Comment (Kars-de-Jong) [inline custom_components/luxtronik/sensor.py:314]: Not sure whether this is needed, I think it is the default for `DEVICE_CLASS_POWER`.

File: custom_components/luxtronik/const.py [hunk 5]
```diff
@@ -371,6 +372,17 @@ class LuxMkTypes(Enum):
         entity_category=EntityCategory.DIAGNOSTIC,
         sensor_key="calculations.ID_WEB_WMZ_Seit",
     ),
+        LuxtronikSensorEntityDescription(
+        key="current_heat_output",
+        # name=f"{text_heat_amount_counter}",
```
Comment (Kars-de-Jong) [inline custom_components/luxtronik/const.py:377]: `name=f"{text_current_heat_output}"`, but it's commented out, doe we still have to add this for new sensors?

**Review by Kars-de-Jong — CHANGES_REQUESTED** [review]
These files should not be committed, `.DS_Store` should be added to the `.gitignore` file:

`.DS_Store`

This will ignore it for all directories.

File: custom_components/luxtronik/const.py (hunk 5 above)
Comment (Crashman1983) [reply custom_components/luxtronik/const.py:377]: doesn't work and is commented out for other sensors as well.

**Comment by Crashman1983** [comment]
All comments fixed, besides naming

File: custom_components/luxtronik/sensor.py [hunk 6]
```diff
@@ -300,6 +301,19 @@ async def async_setup_entry(
             unit_of_measurement=ENERGY_KILO_WATT_HOUR,
             entity_category=EntityCategory.DIAGNOSTIC,
         ),
+        LuxtronikSensor(
```
Comment (BenPru) [inline custom_components/luxtronik/sensor.py:304]: I prefer to check if the sensor is present. My value is always "None".
e.g.:
has_heat_output = luxtronik.get_value("calculations.Heat_Output") is not None
if has_heat_output:

Comment (Crashman1983) [reply custom_components/luxtronik/sensor.py:304]: sorry, just copied the pattern from other sensors. Where should I place the "if"?

Comment (Crashman1983) [reply custom_components/luxtronik/sensor.py]: Think I did it :-)

File: custom_components/luxtronik/const.py (hunk 5 above)
Comment (Kars-de-Jong) [reply custom_components/luxtronik/const.py:378]: OK, but the commented out text is wrong.

**Review by Kars-de-Jong — APPROVED** [review]

**Pull request merged**